packages/marketplace/src/components/pages/client-app-detail/client-app-detail.tsx

```diff
@@ -31,6 +35,24 @@ export const handleInstallAppButtonClick = (setIsVisibleInstallConfirmation: (is
   }
 }
 
+export const handleApplyAppDetailsFromLocalStorage = (dispatch: Dispatch, appid?: string) => () => {
+  try {
+    const appDataString = localStorage.getItem('developer-preview-app')
+    if (!appDataString) {
+      throw 'No app preview'
```

Comment:
does it difference
```
throw new Error('No app preview')
```
Should we redirect user to app detail after throw error ?
![image](https://user-images.githubusercontent.com/22660833/83423244-e6867100-a454-11ea-94fe-15c1eab003a1.png)

Reply:
`throw new Error('No app preview')` can make browser crash in development mode. And no much difference between throw an Error object and throw a text.

`Should we redirect  user to app detail after throw error ?`
Because no way to know the preview appId is correct, so I just show an Info element to user